SwiftStaticLibrary/Public/SwiftStaticLibrary.modulemap

```diff
@@ -0,0 +1,6 @@
+framework module SwiftStaticLibrary {
```

Comment:
This doesn't appear to be necessary (or even doing anything?).  If I remove it, I believe everything still works (still can't import Swift code into Objective-C however).

Reply:
Perhaps this should be `module` instead of `framework module`?